examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,37 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
+:class:`xarray.DataArray` format, and plot it on a map.
+
+The example below shows a Worldview 2 satellite image over
+`Lāhainā, Hawai'i during the August 2023 wildfires
+<https://en.wikipedia.org/wiki/2023_Hawaii_wildfires#L%C4%81hain%C4%81>`_.
+Data is sourced from a Cloud-Optimized GeoTIFF file hosted on
+`OpenAerialMap <https://map.openaerialmap.org>`_ under a
+`CC BY-NC 4.0 <https://creativecommons.org/licenses/by-nc/4.0/>`_ license.
+"""
+import pygmt
+import rioxarray
+
+###############################################################################
+# Read 3-band data from GeoTIFF into an xarray.DataArray object
+image = rioxarray.open_rasterio(
+    filename="https://oin-hotosm.s3.us-east-1.amazonaws.com/64d6a49a19cb3a000147a65b/0/64d6a49a19cb3a000147a65c.tif",
+    overview_level=5,
+)
+# Subset to area of Lāhainā in EPSG:32604 coordinates
+image = image.rio.clip_box(minx=738000, maxx=755000, miny=2300000, maxy=2318000)
+image
+
+###############################################################################
+# Plot the RGB imagery
+fig = pygmt.Figure()
+fig.grdimage(
+    grid=image,
+    projection="x1:100000",
+    frame=[r"WSne+tL@!a\225hain@!a\225, Hawai\047i on 9 Aug 2023", "af"],
```

Comment:
Had to use the composite character `@!a\225` to plot ā following https://docs.generic-mapping-tools.org/6.4/tutorial/session-2.html#plotting-text-strings, because the character ā is in ISO-8859-4, see https://en.wikipedia.org/wiki/%C4%80

Reply:
I think that's fine. Maybe it's possible to avoid that the letters are too close to each other by using another font style?

Reply:
Agree with @michaelgrund, these unequally spaced letters are a bit strange. Looking at the example at https://docs.generic-mapping-tools.org/6.4/tutorial/session-2.html#gmt-tut-10, using the font Times-Roman does not lead to unequally spaced letters.

Reply:
Agree that the spacing looks weird. I've changed the font to Times-Roman at b516c194da89836ecc83b745775ffe41bfb96b88 and it looks a bit better now:

![Lāhainā in Times-Roman font](https://github.com/GenericMappingTools/pygmt/assets/23487320/f78d05f9-3ab7-4ee6-93ff-352ccc2a143e)